content/first-steps.md

```diff
@@ -10,7 +10,7 @@ We'll mostly use TypeScript in the examples we provide, but you can always **swi
 
 #### Prerequisites
 
-Please make sure that [Node.js](https://nodejs.org/) (>= 10.13.0) is installed on your operating system.
+Please make sure that [Node.js](https://nodejs.org/) (a [stable LTS version](https://nodejs.org/en/about/releases/) >= 10.13.0 ) is installed on your operating system.
```

Comment:
Thanks for your contribution! One thing - this sentence doesn't seem to be fully accurate. For example, Node v15 isn't an LTS version, but Nest supports it as well. Any ideas on how we can improve it? cc @jmcdo29

Reply:
It's a tough call, because Node 13 is really the offending version due to the UUID package. Maybe we make a mention of that directly?

Reply:
Yeah, that sounds good to me! cc @fauzulkc

Reply:
> Yeah, that sounds good to me! cc @fauzulkc

@kamilmysliwiec did you mean to tag me? If I am to put in my two cents though - We could explicitly say **unreliable support in Node 13.x**.

Reply:
Oh sorry about that @fauzulkc, I meant to tag @rchod, for some reason Github prompted me your account 😅